plugin/bookmark.vim

```diff
@@ -253,7 +253,9 @@ function! CallDeprecatedCommand(fun, args)
   return call(Fn, a:args)
 endfunction
 
-function! BookmarkModify(diff)
+function! BookmarkModify(diff, ...)
```

Comment:
Can this second (optional?) param be made more explicit? It's hard to read right now as to what this second parameter is about.